commit ba284fcd8361ddd9934acd87b81c107e0e6e5b8e
Author: Caio SBA <caiosba@gmail.com>
Date:   Mon Sep 28 11:27:49 2015 -0300

    Fixing login

diff --git a/www/js/controllers.js b/www/js/controllers.js
index ff5fd08..d17a203 100644
--- a/www/js/controllers.js
+++ b/www/js/controllers.js
@@ -121,10 +121,10 @@ angular.module('confjuvapp.controllers', [])
       .then(function(resp) {
         $scope.closeModal();
         var popup = $ionicPopup.alert({ title: 'Login', template: 'Login efetuado com sucesso!' });
-        $scope.profile = resp.data.person;
+        $scope.profile = resp.data.user.person;
         $scope.setStateAndCityOfProfile();
         popup.then(function() {
-          $scope.loginCallback(resp.data.private_token);
+          $scope.loginCallback(resp.data.user.private_token);
         });
       }, function(err) {
         $scope.closeModal();
